MvvmCross/Core/Core/ViewModels/IMvxCommand.cs

```diff
@@ -5,17 +5,30 @@
 //
 // Project Lead - Stuart Lodge, @slodge, me@slodge.com
 
+using System;
 using System.Windows.Input;
 
 namespace MvvmCross.Core.ViewModels
 {
-    public interface IMvxCommand
-        : ICommand
+    public interface IMvxCommand : ICommand
     {
         void RaiseCanExecuteChanged();
 
         void Execute();
 
         bool CanExecute();
     }
+
+    public interface IMvxCommand<T> : ICommand
```

Comment:
In v4 we had a custom binding that bound `IMvxCommand` to control events, and it worked for both - generic, and non generic commands. In v5 two bindings are needed?